src/zui/ZUIDivider/index.tsx

```diff
@@ -1,13 +1,29 @@
+import { FC } from 'react';
 import { Divider, DividerProps } from '@mui/material';
+import { useTheme } from '@mui/styles';
 
-interface ZUIDividerProps extends DividerProps {}
+type ZUIDividerProps = Partial<
+  Pick<DividerProps, 'flexItem' | 'orientation'>
+> & {
+  /** The variant of the divider. Defaults to 'fullWidth' */
```

Comment:
```suggestion
  /**
   * The variant of the divider. Defaults to 'fullWidth'
   */
```

(See comment in PR #2223)

Reply:
i like it